nmpolicy/internal/resolver/resolver.go

```diff
@@ -198,6 +198,8 @@ func (r *resolver) resolveStringOrCaptureEntryPath() (interface{}, error) {
 		return *r.currentNode.Str, nil
 	} else if r.currentNode.Path != nil {
 		return r.resolveCaptureEntryPath()
+	} else if r.currentNode.Boolean != nil {
```

Comment:
we should change the function name to something else now that we are also resolve booleans.

I think the proper name would be "resolveTerminalOrCapturePath" 

Also the extra mile would be to be able to detect if it's a terminial and return the terminal value as `interface{}`

Reply:
>Also the extra mile would be to be able to detect if it's a terminial and return the terminal value as interface{}

Could you please elaborate on this? I'm not sure I follow.

Reply:
Let's just rename the function